projects/storefrontapp/src/app/spartacus/spartacus-b2c-configuration.module.ts

```diff
@@ -57,6 +58,7 @@ const baseSite = environment.epdVisualization
         },
       },
     }),
+    provideConfigFactory(defaultOccConfigFactory),
```

Comment:
This file belongs to an app, but not lib. And we'd like to make a behavioral change implicitly.
In other words, we don't want to ask customers to put the `provideConfigFactory(defaultOccConfigFactory),` in their app's code, in order for the feature toggle to be effective.

Note: the same applies to the projects/storefrontapp/src/app/spartacus/spartacus-b2b-configuration.module.ts

How we'd like to do it:
- in BaseOccModule [we have](https://github.com/SAP/spartacus/blob/3d43cba2ffc0ac4c06a9e3fa083c64262d9e2d66/projects/core/src/occ/base-occ.module.ts#L30) `provideDefaultConfig(defaultOccConfig),`
- let's replace that line with `provideDefaultConfigFactory(defaultOccConfigFactory),`